pennylane/gradients/finite_difference.py

```diff
@@ -317,9 +313,9 @@ def finite_diff(
         >>> gradient_tapes, fn = qml.gradients.finite_diff(tape)
         >>> gradient_tapes
         [<QuantumTape: wires=[0], params=3>,

```

Comment:
This first one is still displayed as a QuantumTape?

Reply:
Yes, I think because it is the unshifted original tape, whereas the others are created newly :grimacing:

Reply:
Opened a discussion on this [here](https://github.com/PennyLaneAI/pennylane/pull/5596#discussion_r1584661194) (Sorry, I think I was too slow with posting these comments.